example-gto/code/mlem/deploy/staging.mlem

```diff
@@ -0,0 +1,4 @@
+object_type: deployment
+type: heroku
+model: models/churn.pkl
+app_name: gto-example-mlem-app-staging
```

Comment:
we should be generating / adding new lines at the end of all files. This is a good practice.

Reply:
we just use `with open ... as f: dump(obj,f)`

Reply:
@mike0sv I know, but if we expect ppl (and we should) to work with this files, if we expect CLI tools to work properly with these files, we must include newline https://stackoverflow.com/a/729795/298182

Reply:
How does dvc solve this? Probably should create an issue in main repo and continue discussion there

Reply:
that's how for example, regular cat shows this my terminal (zsh):

```
(.venv) √ Projects/example-gto % cat deploy/prod.mlem
object_type: deployment
type: heroku
model: models/churn.pkl
app_name: gto-example-mlem-app-prod%
```

it makes it so that input can stay on the newline still (zsh forces this)